overlord/ifacestate/udevmonitor/udevmon.go

```diff
@@ -38,13 +38,15 @@ type Interface interface {
 
 type DeviceAddedFunc func(device *hotplug.HotplugDeviceInfo)
 type DeviceRemovedFunc func(device *hotplug.HotplugDeviceInfo)
+type EnumerationDoneFunc func()
 
 // Monitor monitors kernel uevents making it possible to find USB devices.
```

Comment:
BTW. this should probably say `hotpluggable devices`, there are more interfaces supporting hotplug than just USB.